### PR TITLE
Add core files.

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

**Review by davidlehn — APPROVED** [review]
Perhaps add some text to the README so people know what this is for.

File: send-token-component.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    _display(property);
+  };
+
+  self.submit = function() {
+    brDidJwtService.sendToken(self.email)
+      .then(function() {
+        _display('success');
+      }).catch(function(err) {
+        brAlertService.add('error', err);
+      });
```
Comment (dlongley) [inline send-token-component.js:45]: Needs a final `.then(function) { $scope.$apply(); })`?

**Review by dlongley — CHANGES_REQUESTED** [review]
One minor thing, then I think this is ok for a first cut of this module.

File: send-token-component.js (hunk 1 above)
Comment (mattcollier) [reply send-token-component.js:45]: br-alerts are being displayed properly and adding `$scope.$apply()` results in an error `$digest already in progress`

File: send-token-component.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<div ng-if="$ctrl.display.form" class="row">
+  <div class="col-md-3"></div>
+  <div class="col-md-6">
+    <div class="text-center">
+      <h3>Email XXXX</h3>
+      <p>Please enter the email address you used when you setup your account so that
+      we may send you a XXXXXXXXXX.</p>
+    </div>
```
Comment (mattcollier) [inline send-token-component.html]: @dlongley Before release, we need to come up with the terminology we want to use here.

**Review by dlongley — APPROVED** [review]

**Pull request merged**